src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,164 @@
+package duke.ui;//package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * The Ui class manages user interface interactions and displays messages to the user.
+ */
+public class Ui {
+    /** The name of the chat bot. */
+    String botName;
+
+    /**
+     * Constructs a Ui object with the specified bot name.
+     *
+     * @param botName the name of the bot
+     */
+    public Ui(String botName) {
+        this.botName = botName;
+    }
+
+    /**
+     * Displays a message with horizontal lines above and below it.
+     *
+     * @param message the message to be displayed
+     */
+    public static void messageWithHorizontalLines(String message) {
+        System.out.println("____________________________________________________________\n" +
+                message + "\n" +
```

Comment:
Line break put should be before operators (i.e. the + should be put on the next line). See [`Layout` section](https://se-education.org/guides/conventions/java/intermediate.html#layout) of Java Coding Standard.